frontend/ui/widget/skimtowidget.lua

```diff
@@ -264,76 +221,86 @@ function SkimToWidget:init()
             self:goToBookmark(page)
         end,
     }
+    self.button_bookmark_toggle = Button:new{
```

Comment:
Random inconsequential nit: this one's an instance member, the others are locals.

Reply:
Yes, just like your `self.current_page_text` , because this one, like yours, go thru `Button:setText()` in `SkimToWidget:update()` (that I tweaked a bit to just use `:text_func()` to avoid duplicating that text building code).

Reply:
Oh, missed that, cool :).